app-service-template/Makefile

```diff
@@ -59,7 +59,7 @@ docker:
 # The test-attribution-txt.sh scripts are required for upstreaming to EdgeX Foundry.
 # TODO: Remove bin folder and reference to script below if NOT upstreaming to EdgeX Foundry.
 test:
-	$(GO) test -coverprofile=coverage.out ./...
+	$(GO) test -coverprofile=coverage.out ./... --race
```

Comment:
Still not consistent with edgex-go. Use variable and position of opinion. Please update to use:

```
$(GO) test $(GOTESTFLAGS) -coverprofile=coverage.out ./...
```

Reply:
https://github.com/edgexfoundry/edgex-go/blob/main/Makefile#L46
https://github.com/edgexfoundry/edgex-go/blob/main/Makefile#L98